lib/spotlight/dor/indexer.rb

```diff
@@ -48,47 +47,52 @@ def solr_document resource
     def resource druid
       Harvestdor::Indexer::Resource.new harvestdor, druid
     end
-    
+
     private
     # def add_something_to_solr_doc sdb, solr_doc
-    #   solr_doc['some_field_name_tsi'] = 'some value'
+    #   solr_doc['some_field_name_tsi'] = 'some value'   # OR
+    #   insert_field solr_doc,'some_multivalued_field_name_ssim',array_of_values
     # end
 
+    def add_donor_tags sdb,solr_doc
```

Comment:
Can there please be a test for this method?  Also, rubocop will prefer space after comma and parens around args (I know the rest of the class isn't doing it.)